src/vs/workbench/contrib/localization/electron-sandbox/localeService.ts

```diff
@@ -26,11 +28,27 @@ export class NativeLocaleService implements ILocaleService {
 		@ILanguagePackService private readonly languagePackService: ILanguagePackService,
 		@IPaneCompositePartService private readonly paneCompositePartService: IPaneCompositePartService,
 		@IExtensionManagementService private readonly extensionManagementService: IExtensionManagementService,
-		@IProgressService private readonly progressService: IProgressService
+		@IProgressService private readonly progressService: IProgressService,
+		@ICommandService private readonly commandService: ICommandService
 	) { }
 
-	private async writeLocaleValue(locale: string | undefined): Promise<void> {
+	private async writeLocaleValue(locale: string | undefined): Promise<boolean> {
+		try {
+			await this.jsonEditingService.read(this.environmentService.argvResource);
+		} catch (error) {
+			this.notificationService.notify({
+				severity: Severity.Error,
+				message: localize('argvInvalid', 'Your argv.json file is not valid. Please open it and fix any parse errors so that your display language can be set properly.'),
+				actions: {
+					primary: [
+						new Action('openArgv', localize('openArgv', "Open argv.json"), undefined, true, () => this.commandService.executeCommand('workbench.action.configureRuntimeArguments')),
```

Comment:
I suggest to use editor service directly here and open the file.

Reply:
done

Reply:
I'm now using the TextFileService. Is that not what you meant by this?

Reply:
I meant opening the argv.json file via editor service instead of relying on an internal command.

Reply:
ah! Done now.